polyglot-clients/README.md

```diff
@@ -1,3 +1,16 @@
 # SDKs for other languages
 
-Language specific client SDKs are maintained at a dedicated [conductor-sdk](https://github.com/conductor-sdk) repository.
\ No newline at end of file
+Language specific client SDKs are maintained at a dedicated [conductor-sdk](https://github.com/conductor-sdk) repository.
+
+
+Check the repository for the latest list, but there are SDK clients for:
+
+## SDK List
+* [Clojure](https://github.com/conductor-sdk/conductor-clojure)
+* [C#](https://github.com/conductor-sdk/conductor-csharp)
+* [Go](https://github.com/conductor-sdk/conductor-go)
+* [Python](https://github.com/conductor-sdk/conductor-python)
+
+
+### In progress (PRs encouraged!)
```

Comment:
Will there be a future PR to change the status? It might be easier to to call out the status in the conductor-sdk repo.

Reply:
Yes. Whenever there is an update, we can do a PR to update this page.